apps/evm/src/pages/Account/NewPage/PerformanceChart/fakeData.ts

```diff
@@ -0,0 +1,602 @@
+export const data = [
+  {
+    apyPercentage: 463.22072070832434,
```

Comment:
I was thinking about this field, what is the meaning of this value? Is it the the sum of the APYs accross all the participating markets at this point in time minus the borrow APYs?

Reply:
It doesn't correspond to anything: I just copied some data from the market history endpoint to build the UI and forgot to remove this property. I'll remove it.